packages/js/product-editor/src/utils/init-block.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { BlockConfiguration, registerBlockType } from '@wordpress/blocks';
+
+interface BlockRepresentation {
```

Comment:
Why an interface instead of a type? Just curious as I've still been gravitating towards types since we use those much more often.

Reply:
This just happens to be what Gutenberg uses, but `type` works fine here as well.

Reply:
Yeah, I know interface works a bit difference in that they inherit from eachother when by the same name. I was just curious since we tend to use `type` in the repo. 👍🏻